app/views/field_slips/_field_slip.html.erb

```diff
@@ -0,0 +1,16 @@
+<div id="<%= dom_id field_slip %>">
+  <p>
+    <strong>Observation:</strong>
+    <%= link_to(field_slip.observation.unique_format_name.t, observation_path(field_slip.observation)) %>
+    <br/>
+    <strong>Project:</strong>
+    <%= link_to_object(field_slip.project) %>
+    <br/>
+    <strong>Code:</strong>
+    <%= field_slip.code %>
+  </p>
+
+
+  <%= render(partial: "shared/matrix_box",
+             locals: { object: field_slip.observation }) %>
+</div>
```

Comment:
I think this is all you need here. (The row width might need adjusting.)
This is from collection_numbers#show
```ruby
<ul class="row list-unstyled mt-3">
  <%= render(partial: "shared/matrix_box", locals: { columns: "col-xs-12" },
             collection: @collection_number.observations, as: :object) %>
</ul>
```

Reply:
Found a solution based on the herbarium record that is working well.